src/components/MenuPanel/MenuPanel.js

```diff
@@ -18,13 +19,24 @@ import MenuPanelAppsLoader from './MenuPanelAppsLoader'
 import NotificationAlert from '../Notifications/NotificationAlert'
 import OrganizationSwitcher from './OrganizationSwitcher/OrganizationSwitcher'
 import AppIcon from '../AppIcon/AppIcon'
+import IconArrow from '../../icons/IconArrow'
 
 const APP_APPS_CENTER = staticApps.get('apps').app
 const APP_HOME = staticApps.get('home').app
 const APP_PERMISSIONS = staticApps.get('permissions').app
 const APP_SETTINGS = staticApps.get('settings').app
 const SHADOW_WIDTH = 15
 
+const systemAppsOpenedState = {
+  key: 'SYSTEM_APPS_OPENED_STATE',
+  isOpen: function() {
+    return window.localStorage.getItem(this.key) || false
```

Comment:
localStorage stores booleans as string. That means that this will return "true" instead of a boolean. 

```suggestion
    const val = window.localStorage.getItem(this.key)
    return typeof val === 'string' ? JSON.parse(val) : false
```

Reply:
Good catch, thanks!